freqtrade/optimize/__init__.py

```diff
@@ -75,6 +131,20 @@ def load_tickerdata_file(
     else:
         return None
 
+    """
+    Call to function to catch if a start or stop date from timerange  is outside
+    range of records in the cached ticker list.
+    This prevents "index out of range" error.
+    """
+    if timerange:
```

Comment:
any particular reason to check `timerange` twice in a row and not combine it with the "if" in line 148?